modules/sqs/main.tf

```diff
@@ -0,0 +1,74 @@
+## Terraform Requirements
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.5.0"
+    }
+  }
+}
+
+
+## AWS Provider Settings
+provider "aws" {
+  region = var.region
+}
+
+## Local Variables
+locals {
+  tags = merge(var.tags, { Deployment = var.prefix })
+}
+
+## copy_files_to_archive_get_message_queue - SQS queue that copy_files_to_archive lambda writes to
+## ====================================================================================================
+resource "aws_sqs_queue" "copy_files_to_archive_get_message_queue" {
```

Comment:
Change the resource name to more closely resemble the name of the queue (less the prefix).

Reply:
changed resource name to staged_recovery_queue